src/Package/MSTest.Sdk/Sdk/Runner/ClassicEngine.targets

```diff
@@ -29,9 +29,13 @@
   <!-- Core -->
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(MicrosoftNETTestSdkVersion)" VersionOverride="$(MicrosoftNETTestSdkVersion)" Sdk="MSTest" />
+    <!-- Most of the times this dependency is not required but we leave the opportunity to align the version of the platform being used -->
```

Comment:
For future readers, consider commenting when it is not needed, and in which situation it might be needed. Because 2 years from now we won't know and will be afraid to remove this.